addons/website/static/tests/tours/website_text_edition.js

```diff
@@ -22,9 +22,14 @@ wTourUtils.registerWebsitePreviewTour('website_text_edition', {
         trigger: '.o_hex_input',
         run: `text_blur ${WEBSITE_MAIN_COLOR}`,
     },
+    {
+        content: "Close colorpicker",
+        trigger: 'we-select[data-color="o-color-1"] .o_we_color_preview',
+    },
     {
         content: "Go to the blocks tab",
         trigger: '.o_we_add_snippet_btn',
+        extra_trigger: 'we-select[data-color="o-color-1"] .o_we_color_preview[style*="background-color: rgb(171, 205, 239)"]',
```

Comment:
> Made test wait for theme color change to be applied

@bso-odoo This makes sense but I actually would prefer this test to stay untouched: this is just hiding the issue. Inputting a hex color and clicking on the blocks tab should work, the test (as it currently is) reveals that it may not. Waiting for the color to visually change should not be required: if something bad could happen doing things while the bundle is not reloaded yet, we should not allow the user to do these things... and also, why did the breaking commit changed that fact? Your commit says "change the tour to avoid the race condition"... but the race condition was not there before the breaking commit, so this is just hiding the race conditon that was recently created... no? :grimacing:

Reply:
@qsm-odoo Fixed a missing `await` and re-nested the mutex call at its former depth. No more change in the test.
Ran a few batches on runbot: all green.